packages/components/src/core/Callout/components/CalloutTitle/index.tsx

```diff
@@ -5,4 +5,7 @@ const CalloutTitle = ({ children }: AlertTitleProps): JSX.Element => {
   return <StyledCalloutTitle>{children}</StyledCalloutTitle>;
 };
 
+// Display name required to ensure Callout expand works correctly.
+CalloutTitle.displayName = "CalloutTitle";
```

Comment:
Can we create a constant for the string, so there's single source of truth 😃?

E.g., `const CALLOUT_TITLE_DISPLAY_NAME = "CalloutTitle` 

`CalloutTitle.displayName = CALLOUT_TITLE_DISPLAY_NAME`


`children[0]?.type?.displayName === CALLOUT_TITLE_DISPLAY_NAME;`